integrations/great_expectations/fraud_batch_data_validation.ipynb

```diff
@@ -0,0 +1,3478 @@
+        "outputId": "534f4af7-f559-4787-f3a1-a8ce2104f3e5"
+      },
+      "id": "3FQTQ7jTNBxh",
+      "execution_count": 22,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "{\n",
+            "  \"expectations\": [],\n",
+            "  \"expectation_suite_name\": \"window_aggs_suite\",\n",
+            "  \"ge_cloud_id\": null,\n",
+            "  \"data_asset_type\": \"Dataset\",\n",
+            "  \"meta\": {\n",
+            "    \"great_expectations_version\": \"0.14.3\"\n",
+            "  }\n",
+            "}\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "expectation_suite_window.add_expectation(validation_result[\"expectation_config\"])\n",
+        "print(expectation_suite_window)"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "ZeM2enlQBAXh",
+        "outputId": "2b0088af-e4d2-4665-cc56-5cb9b983a239"
+      },
+      "id": "ZeM2enlQBAXh",
+      "execution_count": 28,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "{\n",
+            "  \"expectations\": [\n",
+            "    {\n",
+            "      \"expectation_type\": \"expect_column_values_to_be_increasing\",\n",
+            "      \"kwargs\": {\n",
+            "        \"column\": \"datetime\",\n",
+            "        \"strictly\": true,\n",
+            "        \"result_format\": \"BASIC\"\n",
+            "      },\n",
+            "      \"meta\": {}\n",
+            "    }\n",
+            "  ],\n",
+            "  \"expectation_suite_name\": \"window_aggs_suite\",\n",
+            "  \"ge_cloud_id\": null,\n",
+            "  \"data_asset_type\": \"Dataset\",\n",
+            "  \"meta\": {\n",
+            "    \"great_expectations_version\": \"0.14.3\"\n",
+            "  }\n",
+            "}\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Setup Validation On Insert and Upload Data"
+      ],
+      "metadata": {
+        "id": "I3byHYTYEIwN"
+      },
+      "id": "I3byHYTYEIwN"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Register the expectation suite corresponding to a Feature Group with the backend"
+      ],
+      "metadata": {
+        "id": "Dsu6I03nETfg"
+      },
+      "id": "Dsu6I03nETfg"
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# The \"ALWAYS\" ingestion policy inserts data even when validation fails, \n",
+        "# ideal to avoid data loss and rapid prototyping\n",
+        "trans_fg.save_expectation_suite(expectation_suite_trans, validation_ingestion_policy=\"ALWAYS\")\n",
+        "\n",
+        "# The \"STRICT\" ingestion policy only inserts data when data validation succeeds,\n",
```

Comment:
We could add a sneaky comment here that other feature store solutions only do this when retrieving data from the feature store, which doesn't make sense, cause once you have bad data in the feature store, you shouldn't use it anyway.

So kind of explain our approach with "shit in shit out" and that you can prevent it with the "strict" policy.

Reply:
We can but then we should keep an eye on whether it is actually done by other FS, seems impractical to me. I can however emphasise the value of keeping the FS clean

Reply:
Ok emphasizing it is good then.